plugins/cost-insights/src/utils/change.ts

```diff
@@ -24,14 +24,18 @@ import {
 } from '../types';
 import { aggregationSort } from '../utils/sort';
 
-// Used by <CostGrowth /> for displaying status colors
-export function growthOf(amount: number, ratio: number) {
-  if (amount >= EngineerThreshold && ratio >= ChangeThreshold.upper) {
-    return GrowthType.Excess;
-  }
-
-  if (amount >= EngineerThreshold && ratio <= ChangeThreshold.lower) {
-    return GrowthType.Savings;
+// Used for displaying status colors
+export function growthOf(ratio: number, amount?: number) {
+  if (typeof amount === 'number') {
+    if (amount >= EngineerThreshold && ratio >= ChangeThreshold.upper) {
+      return GrowthType.Excess;
+    }
+    if (amount >= EngineerThreshold && ratio <= ChangeThreshold.lower) {
+      return GrowthType.Savings;
+    }
+  } else {
+    if (ratio >= ChangeThreshold.upper) return GrowthType.Excess;
+    if (ratio <= ChangeThreshold.lower) return GrowthType.Savings;
```

Comment:
`growthOf` is used by `CostGrowth` and `CostGrowthIndicator` so that styling logic can be applied consistently. 

For example, if costs are growing they would appear red in the `CostGrowth` component and `CostIndicator` component (with an up arrow). Same logic can be used to derive different behavior/styles.